.gitignore

```diff
@@ -40,6 +40,7 @@ src/config/dash-config.h.in
 src/config/stamp-h1
 share/setup.nsi
 share/qt/Info.plist
+/.vs
```

Comment:
Nit: I know this is trivial, but it is unrelated to the actual change. Generally would be better to keep the PR focused on one specific thing :slightly_smiling_face:

Reply:
yeah, I normally try and avoid that. Didn't think it'd make sense to make a full PR for that tho and couldn't find anyone being too upset about it

Reply:
> Also includes an addition to the .gitnore for us weirdos who use visual studio... Don't judge me.

Just an FYI, this is generally done at a global level, instead of per-repo. For this PR, I think this needs to be removed as [our guidelines](https://github.com/dashpay/dash/blob/develop/CONTRIBUTING.md#pull-request-philosophy) specifically state:

> Patchsets should always be focused. For example, a pull request could add a feature, fix a bug, or refactor code; but not a mixture. Please also avoid super pull requests which attempt to do too much, are overly large, or overly complex as this makes review difficult.

The reason is that it muddies history of features/fixes and makes it harder to track specific issues or revert specific commits -- if we needed to revert this later, we'd also have to revert the `.gitignore` addition.